xml/System.Reflection/PathAssemblyResolver.xml

```diff
@@ -14,8 +14,24 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Class that represents a metadata assembly resolver that uses paths to every assembly that may be loaded.</summary>
+    <remarks>
+      <format type="text/markdown">
+  <![CDATA[
+
+## Remarks
+
+The file name is expected to be the same as the assembly's simple name. Multiple assemblies can exist on disk with the same name but in different directories. A single instance of `PathAssemblyResolver` can be used with multiple <xref:System.Reflection.MetadataAssemblyResolver> instances.
+
+In order for an `AssemblyName` to match to a loaded assembly, `AssemblyName.Name` must be equal (casing ignored).
+
+* If `AssemblyName.PublicKeyToken` is specified, it must be equal.
```

Comment:
There is no AssemblyName.PublicKeyToken property. Perhaps "If a public key token is specified for the <xref:System.Reflection.AssemblyName>…"

Reply:
@steveharter @GrabYourPitchforks can you help clarify what PublicKeyToken this is referring to? I got it from here:
[src/System.Reflection.MetadataLoadContext/src/System/Reflection/PathAssemblyResolver.cs#L18](https://github.com/dotnet/corefx/blob/775347f642ae10badf82bcd0352c9f316093cd6f/src/System.Reflection.MetadataLoadContext/src/System/Reflection/PathAssemblyResolver.cs#L18)

Reply:
@rpetrusha there is a [GetPublicKeyToken method](https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Reflection/AssemblyName.cs#L175). The actual token is private. So I think your comment makes sense. Ill use it.

```suggestion
* If a public key token is specified for the <xref:System.Reflection.AssemblyName>, it must be equal.
```